backend/tournesol/entities/video.py

```diff
@@ -64,13 +64,12 @@ def update_metadata_field(self):
                 self.instance.metadata[metadata_key] = metadata_value
         self.instance.metadata_timestamp = timezone.now()
 
-    def metadata_needs_to_be_refreshed(self):
+    def metadata_needs_to_be_refreshed(self) -> bool:
         """
         Refresh will be executed only if the current metadata
         are older than `VIDEO_METADATA_EXPIRE_SECONDS`.
         The request can be forced with `.refresh_metadata(force=True)`.
         """
-        return self.instance.last_metadata_request_at is None or (
-            timezone.now() - self.instance.last_metadata_request_at
-            >= timedelta(seconds=settings.VIDEO_METADATA_EXPIRE_SECONDS)
-        )
+        return self.instance.last_metadata_request_at is None \
+               or (timezone.now()-self.instance.last_metadata_request_at
+                   >= timedelta(seconds=settings.VIDEO_METADATA_EXPIRE_SECONDS))
```

Comment:
This change makes the CI to fail.

It's caused by `.github/workflows/backend-ci.yml` which runs `./scripts/ci/lint.sh`.

https://github.com/tournesol-app/tournesol/blob/665e2100e4bce4d969b9fee822a264571f2ad86e/.github/workflows/backend-ci.yml#L52

This script run several code quality tools, and will return 1 if one of the checks fails.

You can check the code quality by running this script from the `tournesol/backend` folder before submitting a PR. 

Let's keep the code as it was before the change, as it's " black " formatted*, so everyone can reproduce it by simply running `black` on the file (see https://github.com/psf/black).

```suggestion
        return self.instance.last_metadata_request_at is None or (
            timezone.now() - self.instance.last_metadata_request_at
            >= timedelta(seconds=settings.VIDEO_METADATA_EXPIRE_SECONDS)
        )
```

* note to self: add `black` to the CI : )